pytest_fixtures/subscription_fixtures.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import RHEL_7_MAJOR_VERSION
+from robottelo.rhsso_utils import run_command
+
+
+def unsubscribe():
+    """unregisters a machine from cdn"""
+    run_command('subscription-manager unregister')
```

Comment:
There is a `ContentHost.unregsiter` method that should be used here.

I think it would be good to include an additional call to `clean` in the current function.